metabase/metabase
# Display bookmarks in collections sidebar

In this PR we do display the bookmarks in a Collection's sidebar.

Also, because Bookmarks will now be displayed in that sidebar, we move and rename files so that they better reflect where they are.

Finally, we (I had done this…) had directories of non-container React components inside the the sidebar container directory. 
This way of organizing things did not gain any traction, and I agree it's not ideal — what we have is improvable in other ways. So here I am reverting to the customary directory organization, with non-container components in a different branch of the dir structure than the container.

### How to Test

1. Create a question.
2. Save the question.
3. Visit the question.
4. Open its left sidebar
5. Click on the ribbon icon, the rightmost icon in the list

<img width="400" alt="image" src="https://user-images.githubusercontent.com/380816/157307525-93a61a59-d8aa-42fb-9bac-6e5e1d0b1e6a.png">

It should toggle between black and blue as we click.
Blue means bookmarked.
Black means not bookmarked.

1. Without reloading the page, click on Metabase's icon on the main Nav.
2. Click on `Browse all items`

If you have bookmarked items, they should be rendered on the sidebar.

<img src=https://user-images.githubusercontent.com/380816/157307788-b81ba332-7448-490c-a5f1-f5c831e63d0d.png width=400 />

Refresh the page and you should still see the Bookmarks list.
